backend/infrahub/support/macro.py

```diff
@@ -1,69 +0,0 @@
-from __future__ import annotations
-
-from typing import Any
-
-from jinja2 import TemplateSyntaxError, meta, nodes
-from jinja2.sandbox import SandboxedEnvironment
-
-ALLOWED_FILTERS = [
```

Comment:
Are we still filtering the list of built in filters that are allowed for Jinja2 computed attribute ?

Reply:
Ah yes good catch this is meant to be in the class from the SDK. I will add it.

Reply:
Fixed